### PR TITLE
Html mode for code mirror on HtmlField and HtmlBody standard editor

### PR DESCRIPTION
**Pull request by agriffard**
Related to #5087

### CODE REVIEW

**Comment by agriffard** [comment]
image

**Comment by deanmarcussen** [comment]
LGTM

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField.Edit.cshtml [hunk 1]
```diff
@@ -13,3 +13,21 @@
...
+<script at="Foot">
+    $(function () {
+        var optionsTexArea = document.getElementById('@Html.IdFor(x => x.Html)');
+        var editor = CodeMirror.fromTextArea(optionsTexArea, {
+            lineNumbers: false,
+            lineWrapping: true,
+            matchBrackets: true,
+            mode: { name: "htmlmixed" }
+        });
+        setTimeout(function() { editor.refresh(); },1);
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField.Edit.cshtml]: Why we need to refresh the editor?

Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField.Edit.cshtml]: Without this, the style is only applied the first time the textarea has the focus.

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField.Edit.cshtml]: But we don't have it anywhere else right? Like for liquid editors. What's the difference?

**Pull request merged**